apps/api/src/server/applications/application/application.validators.js

```diff
@@ -103,6 +103,28 @@ const validateExistingMapZoomLevel = async (value) => {
 	}
 };
 
+/**
+ *
+ * @param {string} value
+ * */
+export const validateCaseStage = async (value) => {
+	const isValid = [
+		'pre_application',
+		'draft',
+		'acceptance',
+		'pre_examination',
+		'examination',
+		'recommendation',
+		'decision',
+		'post_decision',
+		'withdrawn'
```

Comment:
Normally I would declare `ApplicationStageType` like

```typescript
const applicationStages = ['draft', 'examination', 'etc'] as const;
type ApplicationStageType = typeof applicationStages[number];
```

but it's not possible to do this in a `.d.ts` file. Not ideal to have a static list of the case stages in multiple places.